automation/check-merged.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -xe
```

Comment:
please drop all -merged modules in favor of https://github.com/kubevirt/bridge-marker/pull/5

Reply:
dropped